src/RapidezServiceProvider.php

```diff
@@ -128,6 +128,7 @@ protected function bootPublishables(): self
 
             $this->publishes([
                 __DIR__ . '/../resources/lang' => resource_path('lang/vendor/rapidez'),
+                __DIR__ . '/../lang'           => resource_path('lang'),
```

Comment:
When it's already published it will not be overwritten right?

Reply:
Correct, although I think this is actually not the right folder (as your other comment mentions)